### PR TITLE
Docs formatting

### PR DESCRIPTION
**Pull request by zeke**
While programmatically parsing the documentation, I noticed a few inconsistencies in the heading sizes across docs. This normalizes all event and method headings to a consistent size, and adds some more detailed info to the styleguide.

### CODE REVIEW

File: docs/api/web-contents.md [hunk 1]
```diff
@@ -1035,15 +1035,15 @@ Detaches the debugger from the `webContents`.
 
 Send given command to the debugging target.
 
-#### Event: 'detach'
+### Event: 'detach'
```
Comment (kevinsawicki) [inline docs/api/web-contents.md]: This event is for the debugger which is already `####` so going back to `###` might make it seem like they refer to `webContents`, instead of `webContents.debugger`.

Comment (zeke) [inline docs/api/web-contents.md]: Good catch. Fixed.

File: docs/api/session.md [hunk 2]
```diff
@@ -50,11 +50,11 @@ const session = require('electron').session;
 const ses = session.fromPartition('persist:name');
 ```
 
-### Instance Events
+## Events
```
Comment (kevinsawicki) [inline docs/api/session.md]: Trying to remember, is reducing these an issue? Is this related to what was discussed at https://github.com/electron/electron/pull/5168#issuecomment-210460464?

Comment (jlord) [inline docs/api/session.md]: Ditto, want to make sure we're still representing it correctly with these updates (per Chengs comments in #5168)

**Comment by zeke** [comment]
Good call, @kevinsawicki. I should have known that, as I saw that issue a while back. I reverted the changes to `session.md`

File: docs-translations/ko-KR/api/session.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ const ses = session.fromPartition('persist:name');
 
 `Session` 객체는 다음과 같은 이벤트를 가지고 있습니다:
 
-#### Event: 'will-download'
+### Event: 'will-download'
```
Comment (zcbenz) [inline docs-translations/ko-KR/api/session.md]: The `Event: 'will-download'` belongs to `Instance Events` section, so I think we should use `####` here?

File: docs-translations/ko-KR/api/session.md [hunk 4]
```diff
@@ -77,7 +77,7 @@ session.defaultSession.on('will-download', (event, item, webContents) => {
 
 `Session` 객체는 다음과 같은 메서드와 속성을 가지고 있습니다:
 
-#### `ses.cookies`
+### `ses.cookies`
```
Comment (zcbenz) [inline docs-translations/ko-KR/api/session.md]: We should probably use `####` here for the same reason above.

Comment (zcbenz) [inline docs-translations/ko-KR/api/session.md]: I just realized these are changes in doc-translations, @zeke I think you forgot to revert the changes there.

File: docs/api/web-contents.md [hunk 5]
```diff
@@ -356,29 +356,29 @@ Returns:
...
-  * `canPaste` Boolean - Wether the renderer believes it can paste.
-  * `canDelete` Boolean - Wether the renderer believes it can delete.
-  * `canSelectAll` Boolean - Wether the renderer believes it can select all.
+  * `canUndo` Boolean - Whether the renderer believes it can undo.
+  * `canRedo` Boolean - Whether the renderer believes it can redo.
+  * `canCut` Boolean - Whether the renderer believes it can cut.
+  * `canCopy` Boolean - Whether the renderer believes it can copy
+  * `canPaste` Boolean - Whether the renderer believes it can paste.
+  * `canDelete` Boolean - Whether the renderer believes it can delete.
+  * `canSelectAll` Boolean - Whether the renderer believes it can select all.
```
Comment (danhp) [inline docs/api/web-contents.md]: Could you also remove the 2 space before the `*` for every property of  `mediaFlags` and `editFlags`? They're causing the [website](http://electron.atom.io/docs/api/web-contents/#event-context-menu) to process the list incorrectly.

**Comment by zeke** [comment]
Good call, @danhp. Fixed.

**Comment by zcbenz** [comment]
👍

**Comment by zeke** [comment]
🎉 thanks @zcbenz
